tests/bdd/scale/test_NAS_T1119.py

```diff
@@ -120,6 +118,27 @@ def the_edit_acl_page_should_open_select_open_for_default_acl_option_select_grou
     driver.find_element_by_xpath('//div[contains(.,"Owner Group:") and @class="control"]//input').click()
     driver.find_element_by_xpath('//div[contains(.,"Owner Group:") and @class="control"]//input').clear()
     driver.find_element_by_xpath('//div[contains(.,"Owner Group:") and @class="control"]//input').send_keys('AD01\\administrator')
+    time.sleep(1)
+    assert wait_on_element(driver, 5, '//span[contains(text(),"Save Access Control List")]', 'clickable')
+    driver.find_element_by_xpath('//span[contains(text(),"Save Access Control List")]').click()
+    time.sleep(2)
+    assert wait_on_element(driver, 5, '//tr[contains(.,"tank_acl_dataset")]//mat-icon[text()="more_vert"]', 'clickable')
+    driver.find_element_by_xpath('//tr[contains(.,"tank_acl_dataset")]//mat-icon[text()="more_vert"]').click()
+    assert wait_on_element(driver, 5, '//button[normalize-space(text())="View Permissions"]')
+    driver.find_element_by_xpath('//button[normalize-space(text())="View Permissions"]').click()
+    time.sleep(2)
+    assert wait_on_element(driver, 5, '//mat-icon[normalize-space(text())="edit"]')
+    driver.find_element_by_xpath('//mat-icon[normalize-space(text())="edit"]').click()
+    assert wait_on_element(driver, 5, '//mat-select[@ix-auto="select__Who"]')
+    driver.find_element_by_xpath('//mat-select[@ix-auto="select__Who"]').click()
+    driver.find_element_by_xpath('//span[contains(text(),"Group")]').click()
+    time.sleep(1)
```

Comment:
instead of sleep add wait on '//input[@data-placeholder="Group"]' to be inputable.